Readme.md

```diff
@@ -148,6 +148,26 @@ collection:
 ./liquid initcollection testdata
 ```
 
+### Nextcloud
+
+Tell liquid we want to run nextcloud and nc_sync jobs in `liquid.ini`:
+
+```ini
+[job:nextcloud]
+template = templates/nextcloud.nomad
```

Comment:
No need for this, just run nextcloud by default. This way travis will try to deploy it too.